engine/pom.xml

```diff
@@ -229,10 +229,10 @@
                             javax.persistence.*;version="[2.0.0,3.0.0)";resolution:=optional,
                             javax.validation.*;version="[1.0.0,2.0.0)",
                             javax.xml.*;version="0",
-                            javax.el.*;version="[2.0.0,4.0.0)",
+                            javax.el.*;version="[2.0.0,4.0.0)";resolution:=optional,
                             org.xml.sax.*;version="0",
                             org.jboss.logging.*;version="[3.1.0,4.0.0)",
-                            com.fasterxml.classmate.*;version="0.8.0",
+                            com.fasterxml.classmate.*;version="[${classmate.version},2.0.0)",
```

Comment:
Isn't the whole point of OSGi that - being modular - different modules can depend on different versions?

I always wished we could somehow make use of ranges in Maven dependency declarations (which are not really meant for modular environments) because of the reasons you state, still we don't as it's essentially impossible to test for it.

If any in the OSGi world I'd expect it to have the same drawbacks than in Maven, but less so a benefit.